tests/serializers/tests.py

```diff
@@ -348,6 +348,18 @@ def test_serialize_proxy_model(self):
         self.assertEqual(base_data, proxy_data.replace('proxy', ''))
         self.assertEqual(base_data, proxy_proxy_data.replace('proxy', ''))
 
+    def test_deserialized_object_class_repr(self):
+        author = Author.objects.create(name='John Doe')
```

Comment:
This seems too complicated, do you really need 3 models? Can you use an in-memory model with its pk set to make the test a bit faster? You could also avoid the need for string interpolation in the expected value if you initialize the model with a pk.